js/config.js

```diff
@@ -14,7 +14,7 @@ _.defaultsDeep(systematic, {
     enable_filename_hashing: true,
   },
   serve: {
-    host: '127.0.0.1',
+    host: '0.0.0.0',  // default of python server. No clear arguments why should be 127.0.0.1.
```

Comment:
The why est dans le commentaire du commit mais peut etre pas assez clair ? j'y dis en substance que dans certains cas tu veux pouvoir tester ton interface depuis un systeme externe (je pense à un browserstack aux tests d'inté, ...) et que du coup  avoir l'UI dispo sur toutes les interfaces par défaut est un plus. Et que le seul argument contre est genre de la "sécuritay" mais là, faut m'expliquer le risque.